docs/ad/api.md

```diff
@@ -389,6 +537,193 @@ POST _opendistro/_anomaly_detection/detectors/results/_search
+              "gt": 0
+            }
+          }
+        },
+        {
+          "nested": {
+            "path": "entity",
+            "query": {
+              "bool": {
+                "must": [
+                  {
+                    "term": {
+                      "entity.value": "i-00f28ec1eb8997685"
+                    }
+                  }
+                ]
+              }
+            }
+          }
+        }
+      ]
+    }
+  },
+  "size": 8,
+  "sort": [
+    {
+      "execution_end_time": {
+        "order": "desc"
+      }
+    }
+  ],
+  "track_total_hits": true
+}
+```
+
+#### Sample response
+
+```json
+{
+  "took": 443,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 7,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": [
+      {
+        "_index": ".opendistro-anomaly-results-history-2020.11.07-1",
+        "_type": "_doc",
+        "_id": "BiItoHUBTpMGN-4KARY5",
+        "_version": 1,
+        "_seq_no": 206,
+        "_primary_term": 1,
+        "_score": null,
+        "_source": {
+          "detector_id": "4CIGoHUBTpMGN-4KzBQg",
+          "schema_version": 2,
+          "anomaly_score": 2.462550517055763,
+          "execution_start_time": 1604710105400,
+          "data_end_time": 1604710094516,
+          "confidence": 0.8246254862573076,
+          "data_start_time": 1604710034516,
+          "feature_data": [
+            {
+              "feature_id": "0Kld3HUBhpHMyt2e_UHn",
+              "feature_name": "latency_max",
+              "data": 3526
+            }
+          ],
+          "execution_end_time": 1604710105401,
+          "anomaly_grade": 0.08045977011494891,
+          "entity": [
+            {
+              "name": "host",
+              "value": "i-00f28ec1eb8997685"
+            }
+          ]
+        },
+        "sort": [
+          1604710105401
+        ]
+      },
+      {
+        "_index": ".opendistro-anomaly-results-history-2020.11.07-1",
+        "_type": "_doc",
+        "_id": "wiImoHUBTpMGN-4KlhXs",
+        "_version": 1,
+        "_seq_no": 156,
+        "_primary_term": 1,
+        "_score": null,
+        "_source": {
+          "detector_id": "4CIGoHUBTpMGN-4KzBQg",
+          "schema_version": 2,
+          "anomaly_score": 4.892453213261217,
+          "execution_start_time": 1604709684971,
+          "data_end_time": 1604709674522,
+          "confidence": 0.8313735633713821,
+          "data_start_time": 1604709614522,
+          "feature_data": [
+            {
+              "feature_id": "0Kld3HUBhpHMyt2e_UHn",
+              "feature_name": "latency_max",
+              "data": 5709
+            }
+          ],
+          "execution_end_time": 1604709684971,
+          "anomaly_grade": 0.06542056074767538,
+          "entity": [
+            {
+              "name": "host",
+              "value": "i-00f28ec1eb8997680"
```

Comment:
Change this entity's value as "i-00f28ec1eb8997685" as we are querying one entity's result in this example.
Same for line 713